site/src/sphinx/advanced-zipkin.rst

```diff
@@ -3,4 +3,71 @@
 Zipkin integration
 ==================
 
-TBW - See :api:`BraveService` and :api:`BraveClient`
+If you want to troubleshoot latency problems in microservice architectures, you will want to use distributed
+tracing system such as `Zipkin <https://zipkin.io/>`_. It gathers timing data and shows which component is
+the problem. Armeria supports `Zipkin <https://zipkin.io/>`_ using
+`Brave <https://github.com/openzipkin/brave/>`_ which is a Java tracing library compatible with it. So, let's
```

Comment:
- How about:
  - Armeria supports distributed tracing via Brave, which is a Java tracing library compatible with Zipkin.
- So, let's -> Let's